src/neo-vm/ExecutionEngine.cs

```diff
@@ -1242,24 +1242,26 @@ private void ExecuteOp(OpCode opcode, ExecutionContext context)
                                             State = VMState.FAULT;
                                             return;
                                         }
-                                    }
                                     break;
+                                    }
```

Comment:
What's the difference? :laughing:

Reply:
usually i use this kind of code:
```
switch
{
 case 1:
  {
  // blablabla
  break;
  }
}
```

instead of 

```
switch
{
 case 1:
  {
  // blablabla
  }
  break;
}
```
There is no difference at all, just i think that is more readable all the code inside the brackets